src/routes/categories/components/CategoryList.js

```diff
@@ -28,9 +42,10 @@ type PropsType = {|
  */
 class CategoryList extends React.Component<PropsType> {
   render () {
-    const {categories, title, content, query, onInternLinkClick} = this.props
+    const {categories, title, thumbnail, content, query, onInternLinkClick} = this.props
     return (
       <div>
+        {thumbnail && <Centering><CategoryIcon src={thumbnail || iconPlaceholder} /></Centering>}
```

Comment:
As far as I can see an icon placeholder will basically never be shown because the `CategoryIcon` only renders when `thumbnail` exists?

Reply:
I think we don't want to display a placeholder icon here. If there is no icon, we shouldn't display one.

Reply:
I agree, just hiding it would be sufficient.